src/Groups/pcgroup.jl

```diff
@@ -365,3 +364,87 @@ function pc_group(c::GAP_Collector)
   end
 end
 
+"""
+    letters(g::Union{PcGroupElem, SubPcGroupElem})
+
+Return the letters of `g` as a list of integers, each entry corresponding to
+a group generator.
+
+# Examples
+```jldoctest
+julia> c = collector(2, Int);
+
+julia> Oscar.set_relative_orders!(c, [2, 3])
+
+julia> Oscar.set_conjugate!(c, 2, 1, [2 => 2])
+
+julia> gg = pc_group(c)
+Pc group of order 6
+
+julia> letters(gg[1]^5*gg[2]^-4)
```

Comment:
This example is not quite obvious for a reader. Better to show the intermediate result, that should make the connection to the output of `letters` "obvious" :-)
```suggestion
julia> x = gg[1]^5*gg[2]^-4
f1*f2^2

julia> letters(x)
```

Reply:
Good idea! I have updated the example in both `syllables` and `letters` to make this output clearer.